src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsworkload/service/JpaBasedSavePersonManagerService.kt

```diff
@@ -49,6 +54,7 @@ class JpaBasedSavePersonManagerService(
     personManagerRepository.save(personManagerEntity)
     telemetryService.trackPersonManagerAllocated(personManagerEntity)
     successUpdater.updatePerson(personManagerEntity.crn, personManagerEntity.uuid, personManagerEntity.createdDate!!)
+    workloadCalculationService.calculate(staff.staffCode, teamCode, staff.probationArea?.code ?: "", gradeMapper.deliusToStaffGrade(staff.staffGrade?.code ?: ""))
```

Comment:
🏆 for most side effects for a method! Makes sense logically to do it in here for now but we could do with thinking about naming